src/views/DownloadPackedOrders.vue

```diff
@@ -349,6 +357,7 @@ export default defineComponent({
           this.selectedData[mapping] = mappingValue[mapping].value
           this.selectedFieldMappings[mapping] = mappingValue[mapping].value
         }
+        this.selectedMappingId = mappingId;
```

Comment:
We should move this outside of the map function as we only need to assign it once and it is not dependent on the mapping values.